src/app/hooks/useClickTrackerHandler/index.test.jsx

```diff
@@ -596,218 +584,216 @@ describe('useClickTrackerHandler', () => {
       expect(mockOptimizelyTrack).toHaveBeenCalledTimes(0);
     });
   });
-});
-
-describe('Click tracking - Reverb', () => {
```

Comment:
All tests have been nested within `useClickTrackerHandler` - which makes the diff look like there are lots of changes, when it's actually just formatting from here to the bottom